VEKT2.CPP

```diff
@@ -31,8 +31,6 @@ void vekt2::forgatas(double alfa) {
 
 double abs(vekt2 a) { return gyok(a.x * a.x + a.y * a.y); }
 
-double absnegyzet(vekt2 a) { return a.x * a.x + a.y * a.y; }
```

Comment:
I don't understand the commit description https://github.com/elmadev/elma-classic/pull/88/commits/f27d95de813e0f75c8e7a97f70b1ce1841efd3da